Samples/VisionARSample/VisionARSample/Base.lproj/LaunchScreen.storyboard

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
Maybe change path to `Samples/VisionAR`?

Reply:
Sure